tools/build_defs/pkg/make_deb.py

```diff
@@ -18,7 +18,11 @@
 from io import BytesIO
 import os
 import os.path
-import StringIO
+from io import BytesIO
```

Comment:
Redundant with the above import (line 18). If importing StringIO from io works, merge that into one line `from io import BytesIO, StringIO`.